src/main/scala/org/broadinstitute/dsde/rawls/util/WithSupport.scala

```diff
@@ -52,3 +52,47 @@ trait MethodWiths {
     }
   }
 }
+
+trait UserWiths {
+  val dataSource: SlickDataSource
+
+  import dataSource.dataAccess.driver.api._
+
+  def withUserInTransaction[T](rawlsUserRef: RawlsUserRef)(op: RawlsUser => Future[T])(implicit executionContext: ExecutionContext): Future[T] = {
```

Comment:
Doug: that is my preferred solution, but it would require refactoring of the handleFutures / toFutureTry code.  I want to remove them but I would like to understand why they exist first.  Do you recall the purpose of this, vs. just handing futures in the normal way?  Are they just relics from a time we were less comfortable with futures?